examples/minimal/src/pages/index.astro

```diff
@@ -0,0 +1,15 @@
+---
+---
```

Comment:
I'm on the fence. Would it be better to not include this at all?

Reply:
the `---` fence? 😆

Reply:
I feel like the frontmatter shouldn't be included if it's empty, similar to how you'd use a markdown file w/o frontmatter

Reply:
I thought about removing the fence, too, and I’m open to that, so I’m glad you asked about this.

I brought some opinions to "minimal". They are —

- A minimal template can be non-instructive.
- A minimal template should be actively helpful.
- A minimal template does not need to be "_minimal_" the "_emptiest possible version_" sense.
- A minimal template should be "_minimal_" in the "_perfect initial commit_" sense.

With those opinions in mind, I **kept the fence**, thinking —

- It’s more likely to stay for adding metadata.
- It can stay or go later. This isn’t a commitment that makes or breaks its usefulness.

So I say _keep it_.

Reply:
I just saw your note right now, @jasikpark. Yea, that’s a good consideration. I defer to y’all. 😄

Reply:
+1 on keeping it!